controllers/controllers.js

```diff
@@ -41,6 +42,16 @@ exports.getComments = (req, res, next) => {
     .catch(next);
 };
 
+exports.patchReview = (req, res, next) => {
+  const { review_id } = req.params
+  const { inc_votes } = req.query
+  return updateVotes(inc_votes, review_id)
+  .then((update) => {
+    res.status(200).send(update[0])
```

Comment:
should return the data on an appropriate key eg review as opposed to just sending update[0]